src/scss/config/_scale.scss

```diff
@@ -36,7 +36,7 @@ $font-sizes: (
   'h1': '#xlarge',
   'h2': '#large',
   'h3': '#medium',
-  'h4': '#small',
+  'h4': 1.1rem,
```

Comment:
Maybe fine, but just to note this also increases the "Past" and "Upcoming" headings on talks.

![Screen Shot 2022-02-23 at 4 24 12 PM](https://user-images.githubusercontent.com/552316/155411017-ad58cc98-409a-4cdf-8948-970d51f3d4b5.jpg)

vs before:

![Screen Shot 2022-02-23 at 4 26 06 PM](https://user-images.githubusercontent.com/552316/155411278-98071614-cb8b-4dd3-8153-f43d679a3b53.jpg)

Reply:
@jgerigmeyer We only used h4 in like 2 places, this is one. I think we could add a class here in this case since the old h4 style is smaller than the body copy in an article.

Reply:
@jgerigmeyer This is fixed in [24c7bd3](https://github.com/oddbird/oddleventy/pull/193/commits/24c7bd3304e4efc868e271d1af76ff6a95bf407e)